### PR TITLE
setup of testing environment with enzyme

### PR DESCRIPTION
**Pull request by Hollister009**
- This PR contains a testing environment setup for unit tests & code coverage

### CODE REVIEW

**Review by ninabondar — COMMENTED** [review]
Thank you for your initiative!

I suggest looking through create-react-app docs, because Epigma is built on top of it. Thus, a lot of configurations, such as testing environment for Jest, stay under the hood. I am not sure that at this stage we really need to add so much configuration as _react-scripts test_ copes with it well.
Please see the structure and setup required to plug Enzyme in with create-react-app: https://create-react-app.dev/docs/running-tests/#initializing-test-environment 
Basically, at this stage tests require adapter config located in src/setupTest.js and some packages: enzyme, enzyme-adapter-react-16 to be installed.
Let's try to stick to YAGNI way of thinking :)
